doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -19,6 +19,40 @@ The following changes are mandatory to make your application work in the same wa
     The setting controls whether the SLM connects automatically to the network on startup.
     You can read and write it using the ``AT#XCARRIER="auto_connect"`` command.
 
+  * The handling of Release Assistance Indication (RAI) socket options has been updated in the ``#XSOCKETOPT`` command.
```

Comment:
The SLM part should be unnecessary as the old values work as well for now, and we will map those to new values under the hood after 2.6.0.

Reply:
As we now have a mechanism in place in SLM to keep such values unchanged (we only change the mapping in SLM, so that the customer-facing interface doesn't change), it may well be that they will never change from the SLM user's point of view.
Though if doing this (which I think is what we want, to limit breaking changes) I don't know whether it actually makes sense to introduce the new values in the documentation unless they work somehow differently than the old ones?

Reply:
I think that the new values are more well thought out than the previous ones So I would like to have them in SLM.

Reply:
Let's have them then, but also keep the other ones forever (without advertising them)?

Reply:
For reference: The section about RAI in SLM is removed from the migration guide for this release. If the old values are removed at a later point (from SLM user perspective), it should be added to the migration guide then.